packages/components/src/ellipsis-menu/test/index.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import { render, fireEvent } from '@testing-library/react';
```

Comment:
Worth mentioning that testing library recommend using the higher level lib `user-event` for a more realistic way to fire events as the user. https://github.com/testing-library/user-event We already have this available as a dependency. not a huge deal in this case as its not used widely in these tests, but can't hurt to form a single approach.